src/Nest/Mapping/Types/Core/String/StringProperty.cs

```diff
@@ -5,7 +5,7 @@
 namespace Nest
 {
 	[JsonObject(MemberSerialization.OptIn)]
-	[Obsolete("Only valid for indices created before Elasticsearch 5.0 and will be removed in the next major version.  For newly created indices, use `text` or `keyword` instead.")]
+	[Obsolete("Only valid for indices created before Elasticsearch 5.0, removed in NEST 6.x.  For newly created indices, use `text` or `keyword` instead.")]
```

Comment:
Maybe `removed in Elasticsearch 6.0`? It's clear that the feature is gone from Elasticsearch and the client reflects this?

Reply:
Makes sense.